box_filter:
    enable: true
    x_min: -2.0
    x_max:  2.0
    y_min: -2.0
    y_max:  2.0
    z_min:  0.0
    z_max:  3.0
voxel_filter:
    enable: true
    x_size: 0.04
    y_size: 0.04
    z_size: 0.01
normal_estimation:
    radius: 0.03